src/solver/simulation/sim_structure_donnees.h

```diff
@@ -70,16 +61,7 @@ typedef struct
 
 typedef struct
 {
-    double* TransitMoyen;   // UNUSED
-    double* TransitMinimum; // UNUSED
-    int* TransitMinimumNo;  // UNUSED
-    double* TransitMaximum; // UNUSED
-    int* TransitMaximumNo;  // UNUSED
-    double* TransitStdDev;  // UNUSED
-    double* TransitMoyenRecalculQuadratique;
-    double* VariablesDualesMoyennes; // UNUSED
-    double* TransitAnnuel;           // UNUSED
-    double* RenteHoraire;            // UNUSED
+    std::vector<double> TransitMoyenRecalculQuadratique;
 } RESULTATS_PAR_INTERCONNEXION;
```

Comment:
Is the struct leftover necessary ?
Can't we just have `std::vector<double> RESULTATS_PAR_INTERCONNEXION` ou `std::vector<double> TransitMoyenRecalculQuadratique`

Reply:
We could have a simple std::vector<std::vector<double>> but I think keep the struct name is clearer, @flomnes what do you think ?

Reply:
No, you could keep only TransitMoyen**Liens**RecalculQuadratique